compiler/record-minmax/driver/Driver.cpp

```diff
@@ -127,8 +136,24 @@ int entry(const int argc, char **argv)
   {
     auto input_data_path = arser.get<std::string>("--input_data");
 
-    // Profile min/max while executing the given input data
-    rmm.profileData(mode, input_data_path, min_percentile, max_percentile);
+    if (input_data_format == "h5" || input_data_format == "hdf5")
+    {
+      // Profile min/max while executing the H5 data
+      rmm.profileData(mode, input_data_path, min_percentile, max_percentile);
+    }
+    // input_data is a text file where the absolute path of each record (a representative data) is
```

Comment:
> absolute path

Actual code just reads the file so I thin relative to CWD would also work.
If your intention is to use absolute path, them we need to check this.
If not, how about changing this to

> file where the file path of each record (a representative data) is

Reply:
> each record

Is this each line of the text file? using word `record` is hard to understand the unit or something...